safe_authenticator/src/apps.rs

```diff
@@ -0,0 +1,236 @@
+// Copyright 2018 MaidSafe.net limited.
```

Comment:
For new files a year should be updated I believe:
```suggestion
// Copyright 2019 MaidSafe.net limited.
```